src/main/java/com/minecolonies/api/colony/IColony.java

```diff
@@ -459,7 +459,20 @@ default List<BlockPos> getWayPoints(@NotNull BlockPos position, @NotNull BlockPo
     void setNameStyle(final String style);
 
     /**
-     * Get the matching citizen name file of the colony .
+     * Check if any civilians have names from a different name pack.
```

Comment:
As explained in the implementation, this shouldn't be part of the Colony, but the CitizenManager